packages/atomic/src/components/search/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -137,8 +138,30 @@ export class AtomicRefineModal implements InitializableComponent {
     this.addFacetColumnStyling(divSlot);
 
     const facets = this.bindings.store.getFacetElements() as BaseFacetElement[];
-
-    const sortedFacets = sortFacetsUsingManager(facets, this.facetManager);
+    const atomicSearchInterface = this.host.closest('atomic-search-interface')!;
```

Comment:
We don't want to explore the whole document, because we don't know if there are other interfaces in the page.
We limit the facet triage to the current interface.